app/views/provider_interface/organisations/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= @provider.name %></h1>
+    <% @ratifying_permissions.each do |_id, permissions| %>
+      <% ratifying_permission, training_permission = permissions %>
+      <h2 class="govuk-heading-m">
+        For courses ratified by <%= ratifying_permission.ratifying_provider.name %> and run by <%= ratifying_permission.training_provider.name %>
+      </h2>
+      <div class="app-banner govuk-!-margin-bottom-4">
+        <div class="app-banner__message">
+          Contact <%= ratifying_permission.training_provider.name %> to change permissions.
+        </div>
+      </div>
+
+      <h2 class="govuk-heading-m">
+        For courses ratifyied by <%= ratifying_permission.ratifying_provider.name %> and run by <%= ratifying_permission.training_provider.name %>
```

Comment:
Typo: `ratifyied`

Reply:
@duncanjbrown good spot - fixed here https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2202/commits/435b376dc918e1839fa1f4b601feab409289e1e2#diff-979630c1c1029745e95aa0c3369a8b4fR16